### PR TITLE
Remove flu shots from bathrooms and kitchen tongs from hardware stores

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
None

#### Purpose of change
Minor itemgroup weirdness I found on my most freshly made savefile

#### Describe the solution
- Split ``harddrugs`` into the old ID (which should present identically ingame) and ``harddrugs_residential`` which is identical but with the flu shot removed. Replaced the old ID with new ones in all non-medical related mapgen palettes and itemgroups. Which is... all but the hospital itemgroup and the modular lab palettes. Yeah vaccines are now rarer
- Removed the kitchen tongs from hardware stores, they seemed to be there with the old assumption that they were metalworking tongs. Wanted to remove them from sewage groups as well, but those groups seem to include some chemical stuff as well so maybe they fit there?

#### Describe alternatives you've considered
- I initially also wanted to get rid of raw lobster chunks laying on campground tables without any plates or anything like that but that's a more complex issue than what I'm willing to tackle right now. Odds are campgrounds might be too trigger happy with what itemgroups they use? Because the itemgroups seem to make sense

#### Additional context
I also should look at what the hell is going on in some zombie drops. Ain't nobody carrying an autoclave or an industrial compressor on their back.

### CODE REVIEW

File: data/json/itemgroups/Drugs_Tobacco_Alcohol/drugs.json [hunk 1]
```diff
@@ -376,9 +376,16 @@
...
-      [ "flu_shot", 5 ],
       [ "syringe", 8 ]
     ]
+  },
+    {
+    "type": "item_group",
+    "id": "harddrugs",
+    "items": [
+      { "group": "harddrugs_residential", "prob": 133 },
+      { "item": "flu_shot", "prob": 5 },
```
Comment (Maleclypse) [inline data/json/itemgroups/Drugs_Tobacco_Alcohol/drugs.json:387]:
```suggestion
      { "item": "flu_shot", "prob": 5 }
```

**Comment by Maleclypse** [comment]
Why are these called "harddrugs"?  These aren't hard drugs.  They are all prescription drugs?

**Comment by Karol1223** [comment]
> Why are these called "harddrugs"? These aren't hard drugs. They are all prescription drugs?

I couldn't tell you, but I'm not too fond of changing an established ID.

**Comment by Maleclypse** [comment]
> > Why are these called "harddrugs"? These aren't hard drugs. They are all prescription drugs?
> 
> I couldn't tell you, but I'm not too fond of changing an established ID.

I’m less concerned about changing an id for an itemgroup but I understand that concern.

**Comment by Cormac1991** [comment]
i work at ace hardware, we sell like 6 types of grilling tongs, hardware stores have a grill section. tmyk

**Pull request merged**